<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DarkBluePlayerWidget</class>
 <widget class="QWidget" name="DarkBluePlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>79</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>79</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="player_Mainframe" native="true">
     <property name="type" stdset="0">
      <string>player_mainframe</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,100">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="coverImage">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>63</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>63</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/border_images/Ressources/no_cover.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="player_InnerFrame">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;b&gt;No Prayer For The Dying&lt;/b&gt;&lt;br&gt;Iron Maiden - Tailgunner</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="playerControls" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;b&gt;Next Song&lt;/b&gt;&lt;br&gt;Iron Maiden - No Prayers For The Dying</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="type" stdset="0">
              <string>nextSong</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ressources.qrc">:/border_images/Ressources/no_cover_small.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="currentTime">
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="seekSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="type" stdset="0">
              <string>seeker</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalTime">
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
